pkg/features/feature.go

```diff
@@ -118,7 +118,7 @@ var (
 		"[Experimental]: Allow Fleet workspace name to be changed on clusters administrated by provisioning v2",
 		false,
 		false,
-		false)
+		true)
```

Comment:
This just ensures that the feature object exists so that UI can find it to enable or disable it.